src/components/button/button.scss

```diff
@@ -44,9 +44,11 @@
 }
 
 // Only flat buttons and icon buttons (not raised or fabs) have a hover style.
-[md-button]:hover, [md-icon-button]:hover {
-  // Use the same visual treatment for hover as for focus.
-  @include md-button-focus();
+[md-button], [md-icon-button] {
+  &:not([disabled]):hover {
```

Comment:
I'm hesitant to get so specific with the rule. Can it be done such that the disabled style is defined after the hover style with the same specificity as to override it?